backend/db_migrations/000008_match.up.sql

```diff
@@ -37,7 +37,7 @@ comment on function app.product_keyword_ingredients(app.product) is 'Given a pro
 
 create or replace function app.ingredient_keyword_products(i app.ingredient) returns SETOF app.product as $$
     SELECT p.*
-    FROM app.product p where i.product_keyword = ANY(p.product_keywords); 
+    FROM app.product p where i.product_keyword ilike ANY(p.product_keywords); 
```

Comment:
As mentioned before, you cannot edit existing migrations. Please create a new migration with this same code. Code is correct.